docs/lib/auth/fragments/common/overview.md

```diff
@@ -19,14 +19,12 @@ The first mode is a common authorization method for REST or GraphQL APIs, while
 
 ## Sign-up and sign-in
 
-For many apps, user sign-up and sign-in is all that is required. Once authenticated the app can talk to an API to access and mutate data. In this case, you can simply create a User Pool by running `amplify add auth` using the Amplify CLI and selecting the default setup. In your application you can use `Amplify.Auth.signUp` and `Amplify.Auth.signIn` (or an Amplify UI component) to complete this process and retrieve tokens. The Amplify client will refresh the tokens calling `Amplify.Auth.fetchAuthSession` if they are no longer valid. 
+For many apps, user sign-up and sign-in is all that is needed. Once authenticated the app can talk to an API to access and mutate data. In this case, you can simply create a User Pool by running `amplify add auth` using the Amplify CLI and selecting the default setup. In your application you can use `Amplify.Auth.signUp` and `Amplify.Auth.signIn` (or an Amplify UI component) to complete this process and retrieve tokens. The Amplify client will refresh the tokens calling `Amplify.Auth.fetchAuthSession` if they are no longer valid. 
```

Comment:
"to complete this process... (to end of paragraph)" -> "and Amplify will handle the rest - retrieving, sending, and refreshing tokens as needed."